spec/models/manageiq/providers/redhat/infra_manager/refresher_3_1_spec.rb

```diff
@@ -515,7 +515,7 @@ def assert_specific_vm_powered_off
       :controller_type => "virtio",
       :present         => true,
       :filename        => "21fc55f7-2775-4fec-8442-fa546e06fabc",
-      :location        => "0",
+      # :location        => "0",
```

Comment:
Why is this spec commented? If it needs to be please state a reason with a TODO.